source/Uncontrolled.test.tsx

```diff
@@ -1,3 +1,53 @@
-test('<Uncontrolled /> works', () => {
-  expect(true).toEqual(true)
+import { fireEvent, render } from '@testing-library/react'
+import React from 'react'
+import Zoom from '../source'
+
+const ZoomImgTest = () => (
+	<Zoom modalLabelText={`Zoom modal`}>
```

Comment:
I need to put a linter in here for this spacing (2 spaces) 😄

Reply:
Oh, I already did. You can see the failures in the build responses below. I recommend installing eslint in your editor and checking out the issues -- you may be able to auto-fix them via your editor, or you can probably run `npm run lint -- --fix`, but I haven't tried that in a hot minute...

Reply:
^ fixed, I'll be sure to run the  `lint -- --fix` cmd moving forward.